src/Exception/ShopURLIsNotReachableException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopware\AppBundle\Exception;
+
+class ShopURLIsNotReachableException extends \RuntimeException
+{
+    public function __construct(string $shopUrl, ?\Throwable $previous = null)
+    {
+        parent::__construct(
+            sprintf(
+                'Shop URL "%s" is not reachable from the internet and cannot be registered.',
```

Comment:
I think `internet` it is not correct. I'd write `is not reachable from the application server`. 
For example, some hosting in China can be reachable from most of internet but not from some AWS zones.

Reply:
Fixed